src/controllers/TidalController.js

```diff
@@ -0,0 +1,25 @@
+var config = {
+  supports: {
+    playpause: true,
+    next: true,
+    previous: true
+  },
+  useLazyObserving: true,
+  playStateClass: 'play-controls__main-button--playing',
+  nextSelector: '.icon-Playback_next',
+  previousSelector: '.icon-Playback_previous'
+}
+
+config.artworkImageSelector = '.player__image-container__image';
+config.frameSelector = '.player__wrapper';
+config.playStateSelector = '.play-controls__main-button--playing';
+config.playSelector = '.play-controls__play';
+config.pauseSelector = '.play-controls__pause';
+config.titleSelector = '.player__heading';
+config.artistSelector = '.player__links';
```

Comment:
why are you defining things down here instead of just including them in the object definition, like you do with nextSelector,previousSelector?